syntaxes/hlasm.tmLanguage.json

```diff
@@ -25,43 +22,70 @@
 				]
 		},
 		"lineTooLong": {
-			"_description": "Assembler lines should only be 72 chars, at 73 the entire line should be in error",
+			"_description": "Assembler lines should only be 80 chars, at 81 the entire line should be in error",
 			"patterns": [
 				{
 					"name": "invalid.illegal.hlasm",
-					"match": "^.{73,}"  
-				}
-			]
-		},
-		"lineComment": {
-			"_description": "All statements that have a * in column 1",
-			"patterns": [
-				{
-					"name": "comment.line.double-slash.hlasm",
-					"match": "^\\*.*$"
+					"match": "^.{81,}"  
 				}
 			]
 		},
-		"continuation": {
-			"_description": "Continuation character that appears in column 72",
+		"parseAllLines": {
+			"_description": "Separate out code from continuation, sequence field, and the rest",
 			"patterns": [
 				{
-					"match": "^(.{71,71})(.)$",
+					"match": "^(.{0,71})(.)(.{0,8})(.*)$",
 					"captures": {
 						"1": {
 							"_description": "Allow syntax hilighting of the rest of the line",
 							"patterns": [
+								{"include": "#lineComment"},
 								{"include": "#asmArgumentLine"},
 								{"include": "#asmLineStart"}
 							]
 						},
 						"2": {
-							"name": "keyword.other.hlasm"
+							"patterns": [
+								{
+									"match": "(\\S)",
+									"name": "keyword.other.hlasm"
+								}
+							]
+						},
+						"3": {
+							"patterns": [
+								{
+									"match": "(\\d{8})",
+									"name": "markup.list.numbered.hlasm"
+								},
+								{
+									"match": "(\\S+)",
+									"name": "markup.list.unnumbered.hlasm"
+								}
+							]
+						},
+						"4": {
+							"patterns": [
+								{
+									"match": "(.*)",
+									"name": "invalid.illegal.hlasm"
+								}
+							]
 						}
+
 					}
 				}
 			]
 		},
+		"lineComment": {
+			"_description": "All statements that have a * in column 1",
+			"patterns": [
+				{
+					"name": "comment.line.double-slash.hlasm",
+					"match": "^(\\*|\\.\\*).*$"
```

Comment:
Is this saying that both of these are comments:

```
* This is a comment
.* This is also a comment
```

Reply:
Yes, the second is a comment within a macro.

Reply:
Alright good do know, @dkelosky can you check if my latest changes handle this. Pretty sure that they don't so it would be another thing to fix.